go/chat/sender.go

```diff
@@ -119,6 +130,45 @@ func (s *BlockingSender) addPrevPointersToMessage(ctx context.Context, msg chat1
 	return updated, nil
 }
 
+// Check that the {ConvID,ConvTriple,TlfName} of msgToSend matches both the ConvID and an existing message from the questionable ConvID.
+// This check is necessary because msgToSend may have been constructed with an untrusted Conv{ID,Triple}.
```

Comment:
Let's throw in a scenario or something here to help visualize what attack we're preventing?